libs/@guardian/libs/src/literalUnionPredicate/literalUnionPredicate.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Create [type predicates][] that narrow primitives
+ * to [literal][] [type unions][].
+ *
+ * [type predicates]: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates
+ * [literal]: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types
+ * [type unions]: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types
+ *
+ * @example
+ * import { createPredicate } from '@guardian/libs';
```

Comment:
```suggestion
 * import { createUnionPredicate } from '@guardian/libs';
```

Reply:
Ah yeah also need to export it from the index… thanks @AshCorr 🤦